LockboxXCUITests/LockboxXCUITests.swift

```diff
@@ -197,18 +161,17 @@ class LockboxXCUITests: BaseTestCase {
         // There should be two matches
         let twoMatches = app.tables.cells.count
```

Comment:
maybe this name could be updated, given that there are far more than two matches :)